static/js/services/live-list.js

```diff
@@ -0,0 +1,246 @@
+// TODO cache activeDom; set it when loading a controller!!
+
+// medic-webapp specific config for LiveList.
+// This service should be invoked once at startup.
+angular.module('inboxServices').factory('LiveListConfig', [
+  '$parse', '$templateCache', 'LiveList',
+  function($parse, $templateCache, LiveList) {
+    // Configure LiveList service
+    return function() {
+      var contactTypes = [ 'district_hospital', 'health_center', 'clinic', 'person' ];
```

Comment:
We copy and paste this everywhere - maybe add an angular constant for the array and then inject it everywhere instead?